.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: Build and test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
+    - run: ./build.sh
```

Comment:
Personally, it would be nice to just put the full build action in the workflow rather than a shell script. We'll get consistent results and it shows up nicely in the UI rather than mixing the two.

Although the downside is that you can't run a workflow locally.

What do other Yoshi repos do? I'd probably say fully using the action and removing the `build.sh` file would be good and we can just create branches/PRs if we'd like to run the build.

Either way, looks good.